src/app/mat-main-nav/mat-main-nav.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, ViewEncapsulation, Input } from "@angular/core";
+import { BreakpointObserver, Breakpoints } from "@angular/cdk/layout";
+import { MatIconRegistry } from "@angular/material/icon";
+import { Observable } from "rxjs";
+import { map, shareReplay } from "rxjs/operators";
+import { DomSanitizer } from "@angular/platform-browser";
+
+@Component({
+  selector: "app-mat-main-nav",
+  templateUrl: "./mat-main-nav.component.html",
+  styleUrls: ["./mat-main-nav.component.scss"],
+  encapsulation: ViewEncapsulation.None
+})
+export class MatMainNavComponent {
+  isHandset$: Observable<boolean> = this.breakpointObserver
+    .observe(Breakpoints.Handset)
+    .pipe(
+      map(result => result.matches),
+      shareReplay()
+    );
+  @Input() skipLinkSelector: string = "maincontent";
+
+  constructor(
+    private breakpointObserver: BreakpointObserver,
+    private matIconRegistry: MatIconRegistry,
+    private domSanitizer: DomSanitizer
+  ) {
+    this.addNHSIcon();
+  }
+
+  addNHSIcon() {
```

Comment:
for my own understanding can i ask why you had to do this?

Reply:
add custom icons to material: https://material.angular.io/components/icon/api#MatIconRegistry